app/templates/frameworks/_agreement_returned_legal.html

```diff
@@ -86,13 +97,21 @@ <h2 class="summary-item-heading">Legal documents</h2>
       <p class="govuk-body">
         <a class="govuk-link" href="{{ url_for('.download_agreement_file', framework_slug=framework.slug, document_name=countersigned_agreement_file) }}" target="_blank">
           {% if framework.frameworkAgreementVersion %}
-            Download your ‘counterpart’ framework agreement signature page
+                {% if is_e_signature_supported_framework %}
+                 Download your signed framework agreement
+                {% else %}
+                  Download your ‘counterpart’ framework agreement signature page
+                {% endif %}
           {% else %}
             Download your countersigned framework agreement
           {% endif %}
         </a>
         <br/>
-        The agreement signed by the Crown Commercial Service
+        {% if e_signature_supported_framework %}
+        This agreement is signed by the Crown Commercial Service.
+        {% else %}
+        The agreement signed by the Crown Commercial Service.
```

Comment:
```suggestion
        The agreement signed by the Crown Commercial Service.
```
Delete this please.

Reply:
@NoraGDS this only appears for non-e-signature frameworks and I guess gives them the difference between the two documents in that case.